cli/disputer.go

```diff
@@ -251,6 +251,9 @@ var disputerStartCmd = &cli.Command{
 				proofsChecked += disputableProofs
 
 				ms, err := makeDisputeWindowedPosts(ctx, api, dl, disputableProofs, fromAddr)
+				if ms == nil {
+					continue
+				}
```

Comment:
Let this continue to the end of the loop. If this is empty, we won't add any messages to dispute, but we still need to add the deadline to the deadline map (caching, IIRC).

Reply:
done